pkg/crd/parser_integration_test.go

```diff
@@ -52,6 +53,18 @@ func packageErrors(pkg *loader.Package, filterKinds ...packages.ErrorKind) error
 	return outErr
 }
 
+func packageErrorsWithMessage(pkg *loader.Package, errMsg string) error {
+	var outErr error
```

Comment:
Do you need a slice of errors to collect all errors? Looks like for now you'll only collect the last package error?
Are we expecting multiple packages to have the same error?

Reply:
In this case, it only needs to collect the last package error to assert just like `func packageErrors()`. We haven't met the case to expect multiple packages having different errors yet.